nextcord/interactions.py

```diff
@@ -1252,3 +1257,43 @@ class InteractionMessage(_InteractionMessageMixin, Message):
     """
 
     pass
+
+
+class MessageInteraction:
+    """Represents a message's interaction data, regardless of application.
+
+    A message's interaction data is a property of a message when the message
+    is a response to an interaction from any bot.
+    .. versionadded:: 2.0
+
+    Attributes
+    -----------
+    id: :class:`int`
+        The interaction's ID.
+    type: :class:`InteractionType`
+        The interaction type.
+    name: :class:`str`
+        The application ID that the interaction was for.
+    user: :class:`User`
+        The user that sent the interaction.
+    data: :class:`dict`
+        The raw data from the interaction.
+    """
+
+    __slots__: Tuple[str, ...] = (
```

Comment:
```suggestion
    __slots__: = (
```
no need to typehint this